app/router.cjsx

```diff
@@ -50,6 +50,7 @@ module.exports =
     <Route path="projects" component={require './pages/projects'} />
     <Route path="projects/:owner/:name" component={require './pages/project'}>
       <IndexRoute component={require './pages/project/home'} />
+      <Route name="disciplines" path="projects/disciplines?" handler={require './pages/projects-disciplines'} />
```

Comment:
This route can be deleted, leaving no changes to this file. Please update by merging https://github.com/zooniverse/Panoptes-Front-End/pull/2369